src/pint/observatory/topo_obs.py

```diff
@@ -335,7 +313,13 @@ def _load_clock_corrections(self):
                 )
             )
 
-    def clock_corrections(self, t, limits="warn"):
+    def clock_corrections(
+        self,
+        t,
```

Comment:
type of t?

Reply:
it is an astropy `Time`. It says in the docstring.